js/main.js

```diff
@@ -173,6 +173,10 @@ jQuery(document).ready(function($) {
 			}
 		}
 
+		var now  = moment();
```

Comment:
You should be able to do this color change entirely with the warning times in the config, shouldn't you?